src/main/views/forbidden.njk

```diff
@@ -0,0 +1,12 @@
+{% extends "layout.njk" %}
+
+{% set title = t('Forbidden | Money Claim') %}
+{% set heading = t('Forbidden') %}
+
+{% block content %}
+  <p>
+    {{ t('You can return to <a href="/legal/">Money Claims</a> and start your claim again.') | safe }}
```

Comment:
I don't think we need a link and a button

Reply:
this is how it across the team